app/src/main/java/org/mozilla/tv/firefox/ext/EngineView.kt

```diff
@@ -105,8 +105,9 @@ fun EngineView.restoreState(state: Bundle) {
     getOrPutExtension(this).webView.restoreState(state)
 }
 
-// TODO replace this with A-C functionality. See #1558
-fun EngineView.currentBackForwardIndex() = this.webView.copyBackForwardList().currentIndex
+fun EngineView.canGoBackTwice(): Boolean {
+    return getOrPutExtension(this).webView.canGoBackOrForward(-2)
```

Comment:
Please always file an AC issue if you require such functionality and we do not provide it yet. I have hopes that one day there's no need for EngineView extensions that access WebView directly anymore. :)

Reply:
We talked about this one when it came up, and filed these two issues. :+1:
https://github.com/mozilla-mobile/firefox-tv/issues/1558
https://github.com/mozilla-mobile/android-components/issues/1340

Reply:
A great! I only remembered the history stack but not the navigation actions. 👍

Reply:
You know, I'm not sure about the preexisting methods.  I'll do an audit and try to find anything without a corresponding A-C issue, and open where needed.

Reply:
@pocmo I opened https://github.com/mozilla-mobile/android-components/issues/1616, let me know if you would like me to break those out into individual issues instead.

Reply:
@Baron-Severin Awesome, thank you! That's fine for now. We'll break those out if needed and once we work on them.